### ISSUE TITLE
Can't reproduce the results reported in the paper

### ISSUE THREAD

**Issue by crazybuthappy** [deepmind/leo#14]
Firstly, thanks a lot for the great job! But I still have several problems as follows.
I ran this code in Titan Xp(with tensorflow==1.13.1), but I only obtained 60.3 test accuracy in 5-way 1-shot setting. And then I switched to the best hyperparameters provided, but only got 60.9 test accuracy with 68.38 validation accuracy, which is also somewhat less than the results reported in the paper.
I tested by setting the 'checkpoint_steps' to 0 and 'evaluation_mode' to True in the runner.py file.
Is something wrong? Would you please give me some hints to reproduce the results reported? @sygi @andreirusu  Thanks a lot!

**Comment by andreirusu** [comment]
Hi, thanks for getting in touch!

Have you tried training on both meta-training and meta-validation sets? We
explain how we do this in accordance with previous SOTA in our paper.

Best wishes,
Andrei

**Comment by crazybuthappy** [comment]
Oh! I see. Thanks a lot for your reply.
